src/Raven.Server/Web/System/AdminDatabasesHandler.cs

```diff
@@ -238,7 +238,94 @@ public Task Get()
                 }
             }
         }
-        
+
+        [RavenAction("/admin/config-expiration-bundle", "POST", "/admin/config-expiration-bundle?name={databaseName:string}")]
+        public async Task ConfigExpirationBundle()
+        {
+            var name = GetQueryStringValueAndAssertIfSingleAndNotEmpty("name");
+            string errorMessage;
+            if (ResourceNameValidator.IsValidResourceName(name, ServerStore.Configuration.Core.DataDirectory.FullPath, out errorMessage) == false)
+                throw new BadRequestException(errorMessage);
+
+            ServerStore.EnsureNotPassive();
+            TransactionOperationContext context;
+            using (ServerStore.ContextPool.AllocateOperationContext(out context))
+            {
+                var configurationJson = await context.ReadForMemoryAsync(RequestBodyStream(), "read-expiration-config");
+                var index = await ServerStore.ModifyDatabaseExpirationBundle(context, name, configurationJson);
```

Comment:
Yes, that would be a cost, but we can use that to moving json of unknown type around.